supervisor/errata_utils.py

```diff
@@ -1,14 +1,15 @@
-from enum import StrEnum
-from functools import cache
 import logging
 import os
+from datetime import datetime, timezone
+from enum import StrEnum
+from functools import cache
 
```

Comment:
Hmm, what are your formatting settings? This seems different from what black wants to do.